bin/serve

```diff
@@ -57,4 +57,10 @@ $server = $factory->launchServer([
 ]);
 
 print "Server running at " . $server->getURL() . " for "  . $path . "...\n";
+
+if (isset($options['open'])) {
+    $CLI_url = escapeshellarg($server->getURL());
+    echo `open $CLI_url`;
```

Comment:
`open` will only work for Mac OS X.

`xdg-open` works for most Linux distos. https://stackoverflow.com/questions/264395/linux-equivalent-of-the-mac-os-x-open-command

Windows uses `start` https://www.computerhope.com/tips/tip188.htm